ui-tests/src/test/java/org/openmrs/reference/AddUserTest.java

```diff
@@ -10,18 +10,25 @@
 
 package org.openmrs.reference;
 
-import org.junit.Ignore;
-import org.openmrs.reference.page.*;
```

Comment:
For quality assurance we rather explicitly import classes since `import org.openmrs.reference.page.*` would avail all the pages which is not really true with the test class in context